core/arch/arm/plat-rd1ae/conf.mk

```diff
@@ -0,0 +1,27 @@
+# RD-1 AE is based on Neoverse V3AE CPU, but there is
+# no compiler support for it yet. Use Neoverse V2 until
+# it becomes available.
+include core/arch/arm/cpu/neoverse-v2.mk
+
+# ARM debugger needs this
+platform-cflags-debug-info = -gdwarf-4
+platform-aflags-debug-info = -gdwarf-4
+
+CFG_ARM64_core ?= y
```

Comment:
If I remember correctly, this should be forced since armv9 doesn't support AArch32 at EL1.

Reply:
Done